### PR TITLE
Updates to provide compatibility with  Fog 2.4.0.

### PR DESCRIPTION
**Pull request by vicomte**
I know a few of us have tackled this problem internally. Is there a way forward to unify this codebase @geemus ?

I have to admit I am messing in here because we are migrating off of RS cloud files/CDN usage so not sure how much more I'll be able to look, but didn't want this to go to waste. I don't use most of the other features but hopefully just the name mangling gets it 90%? Tests seem to be working except for a LoadBalancer arguments error which I haven't tracked down yet. Comments welcome.

### CODE REVIEW

**Comment by geemus** [comment]
@vicomte Thanks! I also haven't done much in/around this in some time. I'm glad this didn't get lost, but am also a little anxious about pulling in a big change without some more active users involved. The renaming is definitely a nice step forward to align with the broader fog direction. I'll give it a look and feedback now.

File: lib/fog/rackspace/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Fog
   module Rackspace
-    VERSION = "0.1.6"
+    VERSION = "0.2.0"
```
Comment (geemus) [inline lib/fog/rackspace/version.rb:3]: If you could leave this out for now I think that would be preferable (I'll change it at release time).

**Review by geemus — COMMENTED** [review]
Looking good to me, other than dropping the version change. I'll see about adding config to run CI in actions so I can maybe help with the error you mentioned also.

**Comment by geemus** [comment]
I've added CI to the main branch, so if you rebase you should get that here as well.

File: lib/fog/rackspace/version.rb (hunk 1 above)
Comment (vicomte) [reply lib/fog/rackspace/version.rb]: reverted this.

**Comment by vicomte** [comment]
As another item, some have resolved this issue by aliasing the old names in their branches to preserve compatibility. Do you prefer that methodology or stay with the breaking change in the main package?

**Comment by geemus** [comment]
In an ideal world, I think it would be great if the gem itself was updated with all the final names that we now expect. If we wanted to have aliases then to still provide backwards compatibility and/or ease of updating I would be open to it though. This approach is (unfortunately) more of a pain now, but my hope is that better consistency across the libs will make it less of a pain moving forward. Does that make sense?

**Pull request closed without merging**